src/code/z_room.c

```diff
@@ -536,13 +536,18 @@ void Room_DrawImage(PlayState* play, Room* room, u32 flags) {
     }
 }
 
-void func_80096FD4(PlayState* play, Room* room) {
+void Room_Init(PlayState* play, Room* room) {
     room->num = -1;
     room->segment = NULL;
 }
 
-u32 func_80096FE8(PlayState* play, RoomContext* roomCtx) {
-    u32 maxRoomSize = 0;
+/**
+ * @brief Allocates memory for rooms and fetches the first room that the player will spawn into.
+ *
```

Comment:
we dont really use "brief" anywhere currently and its weird that this is the only one to get it. I guess its the only comment that is brief. But imo can just remove it